mitodl/open-discussions
# Add django-hijack

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #

#### What's this PR do?
Adds django-hijack so admins can log in as another user.

#### How should this be manually tested?
Log in as someone with access to the admin console.  Go to `/admin/auth/user/`.  Next to each user should be a button `Log in as <username>`.  Click one of those buttons.  You should be able to view the channels that user has access to, create posts and comments as that user, and edit the user's profile.

#### Screenshots (if appropriate)
Admin:
<img width="1152" alt="screen shot 2018-11-28 at 8 54 49 am" src="https://user-images.githubusercontent.com/187676/49156408-53aec880-f2eb-11e8-8c75-b682959b0235.png">

Logged in via Hijack:
<img width="1061" alt="screen shot 2018-11-28 at 8 53 58 am" src="https://user-images.githubusercontent.com/187676/49156409-53aec880-f2eb-11e8-98f7-e28ebbc6d8ac.png">
